easybuild/easyconfigs/f/fmt/fmt-8.1.1-GCCcore-11.2.0.eb

```
@@ -0,0 +1,28 @@
+easyblock = 'CMakeMake'
+
+name = 'fmt'
+version = '8.1.1'
+
+homepage = 'http://fmtlib.net/'
+description = "fmt (formerly cppformat) is an open-source formatting library."
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/fmtlib/fmt/releases/download/%(version)s/']
+sources = ['fmt-%(version)s.zip']
+checksums = ['23778bad8edba12d76e4075da06db591f3b0e3c6c04928ced4a7282ca3400e5d']
+
+builddependencies = [
+    ('CMake', '3.22.1'),
+    ('binutils', '2.37')
+]
+
+separate_build_dir = True
```

this is already the default in `CMakeMake`
```suggestion
separate_build_dir = True
```

---

That is correct. I only updated the previous EC file and overlooked that line. Thanks.